src/style.css

```
@@ -1 +1,82 @@
-/* Styles go here */
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  /* max-width: 1200px;
+  min-width: 1024px; */
+  padding: 0 40px 0 50px;
+  margin: 0 auto;
+  background-color: #fff;
+  box-shadow: 0 2px 4px #e2e7ea;
+}
+
+.nav_list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+}
+
+.nav_item {
+  margin: 0 10px;
+}
+
+.logo {
+  position: relative;
+  bottom: 3px;
+}
+
+.nav_link {
+  
+  display: inline-block;
+  font-weight: 500;
+  font-size: 12px;
+  color: rgb(0, 0, 0);
+  text-decoration: none;
+  line-height: 60px;
+}
+
+.nav_list:last-child {
+  margin-right: 0;
+}
+
+.nav_list:first-child {
+  margin-left: 0;
+}
+
+.nav_link:hover,
+.is-active {
+  color: #00acdc;
+}
+
+/* .nav_link:hover:not(.is-active) {
+  border-bottom: 4px solid #00acdc;
+  border-radius: 3px;
+  position: relative;
+  bottom: 4px;
+} */
+
+/* padding-top: 3px;
+  height: 60px; */
+
+.is-active::after {
+  content: "";
+  display: block;
+  position: relative;
+  bottom: 4px;
```

```suggestion
  bottom: 0;
```
Цей елемент має бути в самому низу nav_link